openlibrary/core/loanstats.py

```diff
@@ -6,18 +6,17 @@
 import time
 import datetime
 import logging
-import simplejson
+import requests
 import web
 from infogami import config
 from .. import i18n
 
-from six.moves import urllib
```

Comment:
My sense is that `urlencode()` is OK to import from urllib.  (That is what requests will do under the hood anyway.)

Reply:
I guess then if `requests.compat` isn't going to be the style - should I replace it with straight `import urllib` because everything is on python3 now?

Reply:
Let's be surgical... `from urllib.parse import urlencode`

Reply:
Ok, that is there now.